src/network.cpp

```diff
@@ -92,7 +96,11 @@ void setupWifi() {
   WiFi.mode(stationMode ? WIFI_AP_STA : WIFI_AP);
   char apName[64];
   sprintf(apName, "Owie-%04X", ESP.getChipId() & 0xFFFF);
-  WiFi.softAP(apName);
+  if (strcmp(Settings->ap_self_password, "") != 0) {
```

Comment:
No need for this check, an empty password passed into this function keeps the WiFi open

Reply:
So it seems like when this check isn't there, the AP name gets all bungled.
Not quite sure why, but my network should be named Owie-C48A, not ESP-DAC48A.
![image](https://user-images.githubusercontent.com/96678052/154776670-9d69d840-7336-421d-8e95-7cd26dcace8d.png)

Reply:
> So it seems like when this check isn't there, the AP name gets all bungled.

Actually, I take this back. The AP name getting bungled and all the following issues seem to be occuring semi-randomly. Investigating now.